frontend/src/components/Logs/RawLogView/index.tsx

```diff
@@ -37,12 +37,19 @@ const convert = new Convert();
 interface RawLogViewProps {
 	isActiveLog?: boolean;
 	isReadOnly?: boolean;
+	isTextOverflowEllipsisDisabled?: boolean;
 	data: ILog;
 	linesPerRow: number;
 }
 
 function RawLogView(props: RawLogViewProps): JSX.Element {
-	const { isActiveLog = false, isReadOnly = false, data, linesPerRow } = props;
+	const {
+		isActiveLog = false,
+		isReadOnly = false,
+		data,
+		linesPerRow,
+		isTextOverflowEllipsisDisabled = false,
```

Comment:
No need to provide `isTextOverflowEllipsisDisabled` as `false`, as it is already mentioned as `false` in `RawLogView.defaultProps`.  Also please remove the false from `isActiveLog` and `isReadOnly` props.